webviews/src/App.tsx

```diff
@@ -264,6 +272,18 @@ function App() {
         <div className="final-setup-group">
           <button className="install-button" onClick={handleSetupGraniteClick} disabled={serverStatus !== ServerStatus.started || !enabled || isKeepExistingConfigSelected}>Setup Granite Code</button>
         </div >
+        {chatModel ? (
```

Comment:
we can detect if there are existing models in ~/.continue/config.json (see https://github.com/redhat-developer/vscode-granite/blob/main/src/configureAssistant.ts#L121). If there are none or there's only 1 model and it's the same as the selected model, then no need to display this warning.

Reply:
Sure. I will change that.